decosjoin/api/decosjoin/decosjoin_connection.py

```diff
@@ -261,6 +261,7 @@ def _transform(self, zaken, user_identifier):  # noqa: C901
                     {"name": "kenteken", "from": "text7", "parser": to_string},
                     {"name": "location", "from": "text8", "parser": to_string},
                     {"name": "status", "from": "title", "parser": to_string},
+                    {"name": "title", "from": "subject1", "parser": to_string}
```

Comment:
Weet je wat voor values doorkomen in subject1? Willen we die überhaupt tonen? Misschien even met Micha overleggen

Reply:
We gebruiken de title property sowieso onder het veldje omschrijving in de frontend. Maar ik ben zeker benieuwd hoe zo'n description eruit ziet. Dus ik overleg verder met Micha

Reply:
Good on ya!